apps/hash-api/src/graphql/create-apollo-server.ts

```diff
@@ -171,6 +172,8 @@ export const createApolloServer = ({
                   );
                 }
               }
+
+              // span?.finish();
```

Comment:
Is this meant to be here?

Reply:
Thanks, that was an old fragment which is not required anymore: bae93498d9c2e458cee6acfe80d71ad0e7415a7b